### PR TITLE
Chronos Evaluator/Metrics API doc

### PR DESCRIPTION
**Pull request by cabuliwallah**
#4356 
You can preview the doc [here](https://yuyin-analytics-zoo.readthedocs.io/en/metrics_doc/doc/PythonAPI/AutoML/automl.html#automl-common-metrics)

### CODE REVIEW

File: pyzoo/zoo/automl/common/metrics.py [hunk 1]
```diff
@@ -180,15 +209,21 @@ def sMDAPE(y_true, y_pred, multioutput='raw_values'):
...
+        \\text{ME} = \\frac{1}{n}\\sum_{t=1}^n  y_t-\\hat{y_t}
+
+    :param y_true: Array-like of shape = (n_samples, \*).
+           Ground truth (correct) target values.
+    :param y_pred: Array-like of shape = (n_samples, \*).
+           Estimated target values.
+    :param multioutput: String in ['raw_values', 'uniform_average']
+
+    :return: Float or ndarray of floats.
+             A non-negative floating point value (the best value is 0.0), or an
```
Comment (TheaperDeng) [inline pyzoo/zoo/automl/common/metrics.py:225]: I think ME is not non-neg? although its best value is 0.

File: pyzoo/zoo/automl/common/metrics.py [hunk 2]
```diff
@@ -350,6 +435,21 @@ class Evaluator(object):
...
+        :param metric: String in ['me', 'mae', 'mse', 'rmse', 'msle', 'r2'
+               , 'mpe', 'mape', 'mspe', 'smape', 'mdape', 'smdape', 'accuracy']
+        :param y_true: Array-like of shape = (n_samples, \*).
+               Ground truth (correct) target values.
+        :param y_pred: Array-like of shape = (n_samples, \*).
+               Estimated target values.
+        :param multioutput: String in ['raw_values', 'uniform_average']
+
+        :return: Float or ndarray of floats.
+                 A non-negative floating point value, or an
```
Comment (TheaperDeng) [inline pyzoo/zoo/automl/common/metrics.py:450]: remove non-negative

File: docs/readthedocs/source/doc/PythonAPI/Chronos/index.rst [hunk 3]
```diff
@@ -9,4 +9,4 @@ Chronos API
     forecasters.rst
     anomaly_detectors.rst
     tsdataset.rst
-   
+    metrics.rst
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/PythonAPI/Chronos/index.rst:12]: let us move this to automl API and call it `automl.common.metrics`

**Review by TheaperDeng — COMMENTED** [review]
others LGTM

**Review by TheaperDeng — APPROVED** [review]
pls run jenkins

**Comment by cabuliwallah** [comment]
Jenkins combined passed:
http://10.112.231.51:18888/job/ZOO-PR-Validation/6204/
http://10.112.231.51:18888/job/ZOO-PR-Validation-Other/6265/

**Pull request merged**